fixture/vcr_cassettes/test_upload_large.json

```diff
@@ -0,0 +1,84 @@
+[
+  {
+    "request": {
+      "body": "{:multipart, [{\"file\", <<0, 0, 0, 32, 102, 116, 121, 112, 105, 115, 111, 109, 0, 0, 2, 0, 105, 115, 111, 109, 105, 115, 111, 50, 97, 118, 99, 49, 109, 112, 52, 49, 0, 0, 58, 107, 109, 111, 111, 118, 0, 0, 0, 108, 109, ...>>, {\"form-data\", [{\"name\", \"file\"}, {\"filename\", \"blob\"}]}, [{\"content-type\", \"application/octet-stream\"}]}, {\"api_key\", \"my_key\"}, {\"signature\", \"85aa2859c260cf11af5beee73f9504fd9ac06875\"}, {\"timestamp\", \"1590470665\"}]}",
```

Comment:
Were these generated? This looks like the result of `IO.inspect` on the `form` variable...

Reply:
yea they should've been, I can try regenerating maybe

Reply:
yea they come out to be the same thing

Reply:
Strange...ok. And the test suite is green? I guess we don't have CI for this fork setup?